string type
string feedbackType
MakiSetting[] settings
int movementTimeMs
